test/format/lrt.c

```diff
@@ -110,8 +110,14 @@ lrt(void *arg)
 			 */
 			testutil_check(session->snapshot(session, "name=test"));
 			__wt_sleep(1, 0);
-			testutil_check(session->begin_transaction(
-			    session, "snapshot=test"));
+			/*
+			 * Keep trying to start a new transaction if it's
+			 * timing out - we know there aren't any resources
+			 * pinned so it should succeed eventually.
+			 */
+			while ((ret = session->begin_transaction(
+			    session, "snapshot=test")) == WT_CACHE_FULL) {}
+			testutil_check(ret);
```

Comment:
KNF, the rest of the code does:
```
while ((ret = 
    session->begin_transaction(session, config)) == WT_CACHE_FULL)
        ;
```

Reply:
That line wrapping doesn't work, but I've changed from `{}` to `;`